app/routes/errors.py

```diff
@@ -75,8 +77,10 @@ def _render_error_page(status_code, template=None, **kwargs):
 
 
 @errors_blueprint.app_errorhandler(400)
-@errors_blueprint.app_errorhandler(BadRequestKeyError)
-def bad_request(exception=None):
+@errors_blueprint.app_errorhandler(BadRequest)
+def bad_request(
+    exception: BadRequest,
```

Comment:
This should still be `BadRequestKeyError` but for type hints it need to be `BadRequest` also because this handles `400` errors.